src/Agent/NewRelic/Profiler/Configuration/InstrumentationConfiguration.h

```diff
@@ -67,6 +67,12 @@ namespace NewRelic { namespace Profiler { namespace Configuration
             ipToFind->MethodName = function->GetFunctionName();
             ipToFind->Parameters = std::unique_ptr<xstring_t>(new xstring_t(methodSignature->ToString(function->GetTokenResolver())));
 
+            if (Strings::AreEqualCaseInsensitive(function->GetAssemblyName(), _X("System.Net.Http"))
```

Comment:
It will be easier for us to understand the conditions in this if statement if the logic was broken out into 2 helper methods: 1. IsHttpClient5OrGreater, and 2. IsSocketsHttpHandlerLessThan5. It would also be helpful to add a comment that these hard-coded exceptions should be temporary and can be removed if version checking is added to the instrumentation xml.